README.md

```diff
@@ -1 +1,270 @@
+
+    ```bash
+    composer exec -- drush user:role:add fedoraadmin admin
+    ```
+
+5. Make the Syn/JWT keys available to our configuration either by (or by some combination of):
+    1. Symlinking the private key to `/opt/islandora/auth/private.key`; or,
+    2. Configuring the location to the private key on the site at
+`http://<your-web-server>/admin/config/system/keys/manage/islandora_rsa_key`
+(where `<your-web-server>` accesses your web server which is configured to serve
+from the starter site's `web/` directory).
+
+6. Run the migrations in the `islandora` migration group to populate the base
+taxonomies, specifying the `--userid` targeting the user with the `fedoraadmin`
+role:
+
+    ```bash
+    composer exec -- drush migrate:import --userid=1 islandora_tags,islandora_defaults_tags
+    ```
+
+This should get you a starter Islandora site with:
+
+* A basic node bundle to represent repository content
+* A handful of media types presentable
+* RDF and JSON-LD mappings for miscellaneous entities to support storage in
+Fedora, Triplestore indexing and client requests.
+
+### Post-installation cleanup
+
+1. Uninstall the database driver modules you are not using; for example, if
+you are using `mysql` to use a MySQL-compatible database, you should be clear to
+uninstall the `pgsql` (PostgreSQL) and `sqlite` (SQLite) modules:
+
+    ```bash
+    composer exec -- drush pm:uninstall pgsql sqlite
+    ```
+
+### Known issues
+
+#### Warnings/errors during installation
+
+Some modules presently have some bad expectations as to the system state when
+`hook_install()` is invoked and as such, some messages are emitted:
+
+```
+$ composer exec -- drush site:install --existing-config --db-url=mysql://user:***@localhost/db
+
+ You are about to:
+ * DROP all tables in your 'db' database.
+
+ Do you want to continue? (yes/no) [yes]:
+ >
+
+ [notice] Starting Drupal installation. This takes a while.
+ [notice] Performed install task: install_select_language
+ [notice] Performed install task: install_select_profile
+ [notice] Performed install task: install_load_profile
+ [notice] Performed install task: install_verify_requirements
+ [notice] Performed install task: install_settings_form
+ [notice] Performed install task: install_verify_database_ready
+ [notice] Performed install task: install_base_system
+ [notice] Performed install task: install_bootstrap_full
+ [error]  The Flysystem driver is missing.
+ [warning] Could not find required jsonld.settings to add default RDF namespaces.
+ [notice] Performed install task: install_config_import_batch
+ [notice] Performed install task: install_config_download_translations
+ [notice] Performed install task: install_config_revert_install_changes
+ [notice] Performed install task: install_configure_form
+ [notice] Performed install task: install_finished
+ [success] Installation complete.  User name: admin  User password: ***
+$
+```
+
+There is two "unexpected" messages there:
```

Comment:
There ```suggestion are``` two "unexpected" messages there:

Reply:
Teehee, yep. Came about from expanding the contraction "There's", which is more common/acceptable in speech despite "There're" being... maybe more correct, but rather rarer... but yeah, will fix.